replica/transition_test.go

```diff
@@ -1 +1,298 @@
 package replica_test
+
+import (
+	"testing/quick"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/renproject/hyperdrive/block"
+	. "github.com/renproject/hyperdrive/replica"
+)
+
+var conf = quick.Config{
+	MaxCount:      256,
+	MaxCountScale: 0,
+	Rand:          nil,
+	Values:        nil,
+}
+
+var _ = Describe("replica TransitionBuffer", func() {
```

Comment:
`Describe("Transition buffer", func() { .. })`

The `"replica"` prefix is implied by the fact that we are in the `package replica_test`.